packages/ui/src/components/ListStats.vue

```diff
@@ -26,9 +27,11 @@
         :index="index"
         :key="player.username"
         :name="player.username"
-        :position="player.position"
+        :position="player.position + 1"
         :score="player.points"
       />
+      <!-- TODO: Uncomment when pagination is fixed -->
+      <!-- <CustomPagination :limit="10" @update-page="updateCurrentPage" /> -->
```

Comment:
I think pagination is already working
```suggestion

      <CustomPagination :limit="10" @update-page="updateCurrentPage" />
```